migration/src/main/java/kin/sdk/migration/core_related/KinAccountCoreImpl.java

```diff
@@ -0,0 +1,197 @@
+package kin.sdk.migration.core_related;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import java.math.BigDecimal;
+import kin.core.Balance;
+import kin.core.EventListener;
+import kin.core.KinAccount;
+import kin.core.ListenerRegistration;
+import kin.core.PaymentInfo;
+import kin.core.Request;
+import kin.core.TransactionId;
+import kin.sdk.migration.interfaces.IBalance;
+import kin.sdk.migration.interfaces.IEventListener;
+import kin.sdk.migration.interfaces.IKinAccount;
+import kin.sdk.migration.interfaces.IListenerRegistration;
+import kin.sdk.migration.interfaces.IPaymentInfo;
+import kin.sdk.migration.interfaces.IRequest;
+import kin.sdk.migration.interfaces.ITransactionId;
+import kin.sdk.migration.exception.AccountNotActivatedException;
+import kin.sdk.migration.exception.AccountNotFoundException;
+import kin.sdk.migration.exception.CryptoException;
+import kin.sdk.migration.exception.InsufficientKinException;
+import kin.sdk.migration.exception.OperationFailedException;
+import kin.sdk.migration.exception.TransactionFailedException;
+
+public class KinAccountCoreImpl implements IKinAccount {
+
+    private final KinAccount kinAccount;
+
+    KinAccountCoreImpl(KinAccount kinAccount) {
+        this.kinAccount = kinAccount;
+    }
+
+
+    @Nullable
+    @Override
+    public String getPublicAddress() {
+        return kinAccount.getPublicAddress();
+    }
+
+    @NonNull
+    @Override
+    public IRequest<ITransactionId> sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount) {
+        return sendTransaction(publicAddress, amount, null);
+    }
+
+    @NonNull
+    @Override
+    public IRequest<ITransactionId> sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount, @Nullable String memo) {
```

Comment:
Correct and not only that but we should change how devplatform use the memo because for the new sdk the memo and app id are acting different.
I will handle it.